lib/ui/home/status_bar_widget.dart

```diff
@@ -84,7 +87,23 @@ class _StatusBarWidgetState extends State<StatusBarWidget> {
               ? _showErrorBanner
                   ? const Text("ente", style: brandStyleMedium)
                   : const SyncStatusWidget()
-              : const Text("ente", style: brandStyleMedium),
+              : GestureDetector(
+                  child: const Text("ente", style: brandStyleMedium),
+                  onTap: () {
+                    showDialogWidget(
```

Comment:
move it behind kDebug or some under Debug section?

Reply:
Was thinking of removing this code after @mnvr is okay with the pressed states and then merging.